cernopendata/modules/fixtures/data/docs/cod-portal-upgrade-31-1-2024/cod-portal-upgrade-31-1-2024.json

```diff
@@ -0,0 +1,25 @@
+[
+  {
+    "author": "CERN",
+    "body": {
+      "content": "cod-portal-upgrade-31-1-2024.md",
+      "format": "md"
+    },
+    "collections": [
+      {
+        "primary": "News"
+      }
+    ],
+    "date_published": "2024-01-31",
+    "featured": 1,
+    "short_description": {
+      "content": "New version of the CERN Open Data Portal deployed"
+    },
+    "slug": "cod-portal-upgrade-31-1-2024",
+    "title": "Upgrade of the CERN Open Data portal",
```

Comment:
I would still prefer something more targeted such as "Improvements to the search functionality" as explained before. It's more descriptive than the generic "upgrade" only.

Reply:
As suggested in the previous comment as well, I've split the initial proposal into two documents: one for the search, one for the upgrade. I would prefer to keep the message of the generic one with the upgrade, since it does more changes than just the search funcionality